### ISSUE TITLE
Saved playlists

### ISSUE THREAD

**Issue by mbhkoay** [meeb/tubesync#132]
Would it be possible to subscribe to your own account's saved playlists?

**Comment by meeb** [comment]
If you don't mind sharing the information, can you paste a link to a saved playlist? It's certainly possible to support but I might need to add validation support for the URL format.

**Comment by mbhkoay** [comment]
Sorry for the misleading question. To clarify, I was wondering if it was possible to say subscribe all the account's saved playlists / created playlist. The idea is that once the user sets tubesync up, the user can forget about it and just add other user's playlist to their own saved playlists, and triggers the download without requiring visiting the webui. 
![image](https://user-images.githubusercontent.com/29805565/126634039-003391be-4cb2-4fc5-8bd4-a7fddc65d095.png)
If it would work together with #28  it's be great.

**Comment by mbhkoay** [comment]
Yes, that would exactly describe what I was trying to achieve, to download saved playlists from other channels, and to download it in such a way that it still maintains the playlists as folders.  

End result would be the videos stored in neat folders, like this
- My own channel
	- Saved playlist 1
	- Saved playlist 2
	- Saved playlist 3

> This only downloads the videos the channel has added themselves though and not videos they have added to playlists from other channels if that's what you mean?

**Comment by meeb** [comment]
Ah right so the only change then would be to create a directory per playlist inside a channel directory. I'll see if that information is available in the metadata, if it is that sounds find to add.

**Comment by meeb** [comment]
Unfortunately I don't think this is possible. The information on playlist isn't returned in the metadata for single videos and a video can be a member of multiple playlists even on the same channel. While TubeSync can de-dedupe videos for "all videos on a channel" by just checking the video ID is unique this doesn't help attempting to sort out what playlist a video might be in, which might be more than one. I can't see a sane way to organise this and keep within sensible crawling / requests to YouTube limits. Even if we wanted to do something like download the video once into the first playlist and then symlink it it over to other playlist directories as per the structure on YouTube it would require recrawling playlists in depth regularly and annoy YouTube's rate limits, as well as weird duplicate videos or symlinks (which break on Windows) everywhere.

A nice idea, but alas likely just isn't technically possible unless someone else has any bright ideas.
